_posts/2015-11-30-translation-process.md

```diff
@@ -0,0 +1,96 @@
+* It’s **more effective** for translation managers who can then invest on this project. The most important point when you request translations is to give context to your translators: this will ensure the quality of your translations. For example, we have the word “Table” in our app: if the person translating does not even know we’re a football application, he\she might translate it as a kitchen table! With Crowdin, we have the possibility to add some context to new copies by entering some notes or even by attaching screenshots underneath them.
+* At last, having a reference project forces you to be as much **consistent** as possible when it comes to define new copies across your platforms. Consistency is a rule when you have 4 different platforms and in many cases like this one it also results in a gain of time and money having translators translating one copy instead of four.
+
+
+
+### Integration in the translation process
+
+In our set-up, developers are responsible for their own, platform-specific, project on Crowdin. They’re responsible for keeping their root file up-to-date and they’re responsible for informing the translation managers if they are missing some copies at the approach of a release.
+
+As soon as translators have translated the new copies and developers have updated their root file (with the new copies in English), the translations managers use the Crowdin “auto-translate” feature that populate the newly collected translations from the reference project to all platform-specific projects.
+
+In practice, this step requires, on Crowdin, a 100% match between the projects’ root files copies. For example, translations for the copy “Let’s start!” will not be applied to the Android project if Android uses the copy “Let’s start” (without an exclamation mark). In some cases, it is a good way to enforce all platforms to be consistent. In some others, it causes important limitations: some platforms have restrictions in regards to the way they declare their variables in strings. This aspect forces translation managers to manually adapt the translations for those copies in platforms-specific projects.
+
+At the end, once translations are all populated in platform-specific projects, developers can download all the resulting translations files. They can do this either in a manual manner, by downloading a zip file including all the translation files, or in a more automatic way, by integrating Crowdin API in their development project and, with one command line, upload updates or download translations.
+
+
+
+
+## Summary and conclusion
+
+In a nutshell, the process is as follow:
+
+1. A Feature owner defines a new feature.
+2. He\she asks a native speaker to approve all the copies he defined.
```

Comment:
`He\she` I'd use a slash instead of a backslash https://en.wikipedia.org/wiki/Slash_(punctuation)#Usage